jetty9/pom.xml

```diff
@@ -27,12 +27,6 @@
       <version>${project.version}</version>
       <type>tar.gz</type>
     </dependency>
-    <dependency>
```

Comment:
This is an unrelated fix.  Should be in async-support branch anyway to avoid unneccessary transitive dependencies.